src/lib/map.js

```diff
@@ -123,3 +123,12 @@ module.exports.getRoomScore = function (toRoom, fromRoom, opts = {}) {
   }
   return scores.WEIGHT_NEUTRAL ? scores.WEIGHT_NEUTRAL : PATH_WEIGHT_NEUTRAL
 }
+
+// Our own implementation of depricated function
+module.exports.isRoomAvailable = function (roomName) {
+  const homeRoomStatus = Game.map.getRoomStatus(_.values(Game.spawns)[0].room.name).status
```

Comment:
```suggestion
  const homeRoomStatus = Game.map.getRoomStatus(Object.values(Game.spawns)[0].room.name).status
```
`Object.values()` is a bit faster.